src/ArticleSection.jsx

```diff
@@ -0,0 +1,23 @@
+import ArticleList from "./ArticleList";
+import ArticleVotes from "./ArticleVotes";
+import { useState, useEffect } from "react";
+import axios from "axios";
+const ArticleSection = () => {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    axios
+      .get("https://nc-news-api-mq3o.onrender.com/api/articles")
+      .then((response) => {
+        setArticles(response.data.articles);
```

Comment:
Should add a loading pattern, so users with slow internet don't just see a blank screen while waiting for the articles to load, if you're not sure how to do this checkout the bottom of the useEffect notes https://notes.northcoders.com/courses/js-front-end/useEffect